docs/index.html

```diff
@@ -568,4 +568,4 @@ <h3>why the Q &amp; A?</h3>
   }
 }
 </script></body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
and this should be a second whitespace issue which might create problems with the CI